pkg/sdn/api/validation/validation.go

```diff
@@ -85,6 +86,10 @@ func ValidateClusterNetworkUpdate(obj *sdnapi.ClusterNetwork, old *sdnapi.Cluste
 func ValidateHostSubnet(hs *sdnapi.HostSubnet) field.ErrorList {
 	allErrs := validation.ValidateObjectMeta(&hs.ObjectMeta, false, path.ValidatePathSegmentName, field.NewPath("metadata"))
 
+	if hs.Host != hs.Name {
```

Comment:
I do not have a strong opinion on if we should or should not default this value.  All existing code should be setting both of them to the same thing, so I am leaning towards just leaving this as-is.